src/main/kotlin/calculator/Calculator.kt

```diff
@@ -0,0 +1,27 @@
+package calculator
+
+import java.util.LinkedList
+import java.util.Queue
+
+class Calculator(
+    private val convertor: Convertor = Convertor()
+) {
+    private val signs: Queue<MathematicalSign> = LinkedList()
+    private val numbers: MutableList<Int> = mutableListOf()
+
+    /**
+     * (순서를 무시하는) 사칙연산 함수
+     */
+    fun arithmeticOperation(numberSentence: String?): Int {
+        val inputList = convertor.splitNumberSentenceToList(numberSentence)
+
+        inputList.forEachIndexed { index, input ->
+            when (index % 2) {
+                0 -> numbers.add(convertor.getNumber(input))
+                1 -> signs.add(convertor.getMathematicalSigns(input))
```

Comment:
```suggestion
            when (index % 2) {
                NUMBER_INDEX -> numbers.add(convertor.getNumber(input))
                OPERATOR_INDEX -> signs.add(convertor.getMathematicalSigns(input))
```
조금 더 의미있는 숫자로 변경해 주는것 도 좋을것 같네요!
[매직넘버](https://javabom.tistory.com/28?category=838674)